symfony.lock

```diff
@@ -163,7 +163,7 @@
         "version": "2.5.2"
     },
     "php": {
-        "version": "7.3"
+        "version": "7.2"
```

Comment:
You really should have a uniform version of php among you. The symfony.lock php version shouldn't change without a real decision to change the php version of the project